OpenRA.Mods.Common/Widgets/SimpleLabelWidget.cs

```diff
@@ -0,0 +1,22 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2017 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+namespace OpenRA.Mods.Common.Widgets
+{
+	public class SimpleLabelWidget : LabelWithTooltipWidget
```

Comment:
LabelWithTooltipWidget has world parameter in ctor, but it isn't used. So it could be removed and I could use directly LabelWithTooltipWidget to avoid SimpleLabelWidget .